io.asgardeo.java.oidc.sdk/src/main/java/io/asgardeo/java/oidc/sdk/config/FileBasedOIDCConfigProvider.java

```diff
@@ -73,6 +73,24 @@ private void initConfig(Properties properties) throws SSOAgentClientException {
         JWSAlgorithm jwsAlgorithm =
                 StringUtils.isNotBlank(properties.getProperty(SSOAgentConstants.ID_TOKEN_SIGN_ALG)) ?
                         new JWSAlgorithm(properties.getProperty(SSOAgentConstants.ID_TOKEN_SIGN_ALG)) : null;
+        try {
+            int httpConnectTimeout =
```

Comment:
Have a private method for the config resolve part. Would make the code neat

Reply:
I mean one to extract the below logic

```
StringUtils.isNotBlank(properties.getProperty(SSOAgentConstants.HTTP_CONNECT_TIMEOUT)) ?
                            Integer.parseInt(properties.getProperty(SSOAgentConstants.HTTP_CONNECT_TIMEOUT)) :
                            SSOAgentConstants.DEFAULT_HTTP_CONNECT_TIMEOUT
```

Reply:
Thanks for the suggestion. Added with https://github.com/asgardeo/asgardeo-java-oidc-sdk/pull/24/commits/14ee73334fe765cd4874057b50182dd4cc7b18d9.